tensorflow/lite/micro/micro_profiler.h

```diff
@@ -72,6 +76,14 @@ class MicroProfiler {
   uint32_t end_ticks_[kMaxEvents];
   int num_events_ = 0;
 
+  struct ticks_per_tag {
+    const char* tag;
+    uint32_t ticks;
+  };
+  ticks_per_tag total_ticks_per_tag[kMaxEvents];
```

Comment:
add a comment that in practice the number of unique tags will be less than the number of events but it is theoretically possible for each event to be unique and hence we allow for the TicksPerTag  array to have kMaxEvents entrires.

Reply:
Done. Thank you for the suggestion!